app.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Workarea Commerce",
+  "addons": [
+    {
+      "plan": "mongolab:sandbox",
+      "as": ["WORKAREA_MONGOID", "WORKAREA_MONGOID_METRICS"]
+    },
+    {
+      "plan": "heroku-redis:hobby-dev",
+      "as": ["WORKAREA_REDIS", "WORKAREA_REDIS_CACHE"]
+    }
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/heroku/heroku-buildpack-ruby"
+    },
+  ],
+  "env": {
+    "SECRET_KEY_BASE": {
+      "generator": "secret",
+      "required": true
+    },
+    "WORKAREA_ELASTICSEARCH_URL": {
+      "required": true
+    },
+    "WORKAREA_S3_ACCESS_KEY_ID": {
+      "required": true
+    },
+    "WORKAREA_S3_SECRET_ACCESS_KEY": {
+      "required": true
+    },
+    "WORKAREA_S3_BUCKET_NAME": {
+      "required": true
+    },
+    "HEROKU_APP_NAME": {
+      "required": "true"
+    },
+    "HEROKU_PARENT_APP_NAME": {
+      "required": "true"
+    }
+  },
+  "scripts": {
```

Comment:
the following two scripts set up the DB when created and remove Elasticsearch indexes when destroyed. this is to reduce load on the shared ES server that all review apps will be connected to.